packages/Krumbs/Krumbs.vue

```diff
@@ -52,6 +59,12 @@ export default {
   border-radius: 0.25rem;
 }
 
+.krumb-item {
```

Comment:
can you apply the truncate helper here?
https://kongponents.netlify.app/style-guide/utilities.html#general-helpers

Reply:
yes!